README.md

```diff
@@ -18,6 +18,9 @@ environment, hipCUB uses the rocPRIM library as the backend.  However, on CUDA p
 * For NVIDIA GPUs:
   * CUDA Toolkit
   * CUB library (automatically downloaded and by CMake script)
+* Python 3.6 or higher (HIP on Windows only)
```

Comment:
I the requirement is for the build scripts only. We have no hard python requirements.

Reply:
good point. i'll edit the READMEs to be more explicit that python is for the build scripts.